docs/block-api.md

```diff
@@ -396,14 +408,16 @@ align: true,
 align: [ 'left', 'right', 'full' ],
 ```
 
-- `alignWide` (default `true`): Gutenberg allows to enable [wide alignment](../docs/extensibility/theme-support.md#wide-alignment) for your theme. To disable this behavior for a single block, set this flag to `false`.
+- `alignWide` (default `true`): This property allows to enable [wide alignment](../docs/extensibility/theme-support.md#wide-alignment) for your theme. To disable this behavior for a single block, set this flag to `false`.
 
 ```js
 // Remove the support for wide alignment.
 alignWide: false,
 ```
 
-- `anchor` (default `false`): Anchors let you link directly to a specific block on a page. This property adds a field to define an id for the block and a button to copy the direct link.
+- `anchor` (default `false`): This property adds a field to define an id for the block to let you link directly to a specific block on a page.
```

Comment:
The first sentence here seemed helpful, I don't think it needs to go. 🤷‍♂️

Reply:
After re-reading have restored the original text that was  better and clear